src/main/java/seedu/duke/commands/SelectPatientCommand.java

```diff
@@ -45,6 +45,10 @@ public SelectPatientCommand(int index, State state) {
      */
     @Override
     public CommandResult execute() {
+        //check for invalid input. -2 is result from non integer input
```

Comment:
We should not introduce new variables as it may cause more errors than expected.